default/templates/default/to-spreadsheet.html

```diff
@@ -110,13 +113,23 @@
                             $('.response-success').removeClass('hidden');
                             })
                         .fail(function(){
+                            $('.response-fail').html(
+                                '{% trans "An error has occurred! Please verify that all your files are valid OCDS JSON, and try again in a few minutes." %}'
+                            );
                             $('.response-fail').removeClass('hidden');
                             })
                         .always(function(){
                             $('#processing-modal').modal('hide');
                             });
                 })
-                .always(function(){
+                .fail(function(jqXHR, textStatus, errorThrown){
+                    $('.response-fail').html(
+                        '{% trans "<b>Error in file uploaded</b>: " %}'
```

Comment:
Should this be "The file could not be uploaded"?

Reply:
It might be that the file wasn't uploaded, but it could be another validation error as well. I'll test and add more explicit messages here.